public/assets/activated.svg

```diff
@@ -1,4 +1,4 @@
 <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M9 5L6.25 9L5 7.18182" stroke="#37AD4B" stroke-linecap="round" stroke-linejoin="round"/>
-<circle cx="7" cy="7" r="6" stroke="#37AD4B"/>
+<path d="M9 5L6.25 9L5 7.18182" stroke="#FF9B3F" stroke-linecap="round" stroke-linejoin="round"/>
+<circle cx="7" cy="7" r="6" stroke="#FF9B3F"/>
```

Comment:
Did you check that this change does not impact protection screens in an unwanted way?

Reply:
OK